### PR TITLE
Added automatic dev environment build

### PR DESCRIPTION
**Pull request by dmnunez1993**
Added scripts for automated build and run docker environment

### CODE REVIEW

File: docker/run [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    return subprocess.check_output('git rev-parse --show-toplevel'.split()).strip()
+
+
+def run_dev_environment(command):
+    user = os.getlogin()
+    uid = os.getuid()
+    docker_args = []
+    image_name = 'create_ros_kinetic_gazebo9'
+
+    docker_args.append("-it")
```
Comment (eborghi10) [inline docker/run:45]: Have you ever used [this](https://docker-py.readthedocs.io/en/stable/)?

Comment (dmnunez1993) [reply docker/run:45]: So far I had never encountered that. Maybe it's even easier to handler docker from python like that

**Comment by eborghi10** [comment]
Added modification into [this branch](https://github.com/RoboticaUtnFrba/create_autonomy/compare/kinetic-devel...RoboticaUtnFrba:dmnunez1993/automated_dev_environment_build).

**Comment by eborghi10** [comment]
Closing this in favor of #132

**Pull request closed without merging**